docs/en/advanced/Scalability-Tuning.md

```diff
@@ -97,20 +107,14 @@ alluxio.user.rpc.retry.base.sleep=1s
 The retry duration and sleep duration should be increased if frequent timeouts are observed
 when a client attempts to communicate with the Alluxio master.
 
-### Thread Pool Size
-
-On a single client, the number of threads connecting to the master is configured by the
```

Comment:
The master client pool size no longer relate to number of connections to master. On master side the number of threads used are also not related to the number of connections.

Reply:
Talked with @ggezer and we will remove this section for now until we understand better what is the impact of this property.